backend/init_database.js

```diff
@@ -9,13 +9,37 @@ if (!fs.existsSync(dbpath)) {
   console.log('New file created')
 }
 
+function validPoem (poem) {
+  if (typeof poem.poem !== 'string') {
+    console.log('error: the poem should be a string')
+    return 0
```

Comment:
i ran some checks and poem.text comes out as an undefined attribute of the poem, but poem.poem returns the poem. i think this may be a discrepancy in the rest-api documentation?

Reply:
@armiesingh what sort of "checks" did u run? 

when i run the server i can't submit a poem through the index page nor through the `test-validation.html` page i created (w/the test code i have above in my first code review comment)

if there is some other discrepancy somewhere we need to resolve it, b/c the code u have right now doesn't work w/the rest of the app, this is the function that submits the user's input:
https://github.com/net-art-uchicago/cta-file-sharing/blob/235b371bfe9986e425e9fa4f8b22c3a0e1abdcb6/frontend/js/firstPage.js#L55-L67

notice how the object being sent has a `.text` parameter, not a `.poem` parameter. Also notice how there is no `.lat` paramter (which u're checking for twice by the way) there's a `.location` parameter which is an array, so `.location[0]` is the latitude && `.location[1]` is the longitude

Reply:
I change the parameters and see what you mean now. The test file we had written earlier was creating the incorrect poem struct. What I'm getting is a "404 add poem not found' error in the console. I will push the new, corrected code.